common4j/src/main/com/microsoft/identity/common/java/crypto/SigningAlgorithm.java

```diff
@@ -29,31 +29,14 @@
  * levels.
```

Comment:
Probably worth expanding on this comment here about why the ones selected diverge from the ones presented in https://developer.android.com/training/articles/keystore

Reply:
ACK

Reply:
f58509a9f6d90963e633380141e45059f539fc98